pom.xml

```diff
@@ -46,7 +46,7 @@
   </distributionManagement>
 
   <properties>
-    <jenkins.version>2.121.2</jenkins.version>
+    <jenkins.version>2.145</jenkins.version> <!-- to pick up https://github.com/jenkinsci/jenkins/pull/3662 -->
```

Comment:
Since that fix is not in any LTS line yet, I think it would be good to release what we currently have in master for anyone using JEP-210 in recent LTS lines, and then we can release this afterwards as a complete fix for future LTS lines.